### ISSUE TITLE
print dataset during transformation

### ISSUE THREAD

**Issue by ifenty** [ECCO-GROUP/ECCO-pipeline#44]
During transformation the message to the screen reads:

```[INFO] 2023-10-16 10:22:15,509 - Transforming 2021-05-16T00:00:00Z to ECCO_llc90```

But there are so many messages like this I can't tell which dataset is being processed.  Better if the message was like:

```[INFO] 2023-10-16 10:22:15,509 - Transforming``` **DATASET XYZ** ```2021-05-16T00:00:00Z** to ECCO_llc90```

**Comment by kevinmarlis** [comment]
Yes, that's part of the problem with multiprocessing, it's very easy to lose track of what's what. Let's track the improvements you'd like to the messaging in this issue.
